src/ui/widgets/Input/input.tsx

```diff
@@ -32,73 +33,149 @@ export interface InputProps {
   onClick: (event: React.MouseEvent<HTMLInputElement>) => void;
   font?: Font;
   textAlign?: "left" | "center" | "right";
+  horizontalAlignment?: number;
+  verticalAlignment?: number;
 }
 
+const InputWidgetProps = {
+  ...PVWidgetPropType,
+  font: FontPropOpt,
+  foregroundColor: ColorPropOpt,
+  backgroundColor: ColorPropOpt,
+  transparent: BoolPropOpt,
+  alarmSensitive: BoolPropOpt,
+  enabled: BoolPropOpt,
+  textAlign: ChoicePropOpt(["left", "center", "right"]),
+  horizontalAlignment: FloatPropOpt,
+  verticalAlignment: FloatPropOpt
+};
+
+const TextField = styled(MuiTextField)({
+  "&.MuiFormControl-root": {
+    height: "100%",
+    width: "100%",
+    display: "block"
+  },
+  "& .MuiInputBase-root": {
+    height: "100%",
+    width: "100%"
+  },
+  "& .MuiOutlinedInput-root.Mui-focused .MuiOutlinedInput-notchedOutline": {
+    border: "2",
+    borderColor: "#000000"
+  },
+  "&:hover .MuiOutlinedInput-root .MuiOutlinedInput-notchedOutline": {
+    border: "1",
+    borderColor: "#000000"
+  }
+});
+
 export const SmartInputComponent = (
   props: PVInputComponent & {
```

Comment:
I think this was already being done an odd way, but it's worth changing the way the props are handled so it's similar to  Readback or Label e.g.

```
export type InputComponentProps = InferWidgetProps<typeof InputProps> &
  PVComponent;

// inside Component definition
props: InputComponentProps
```

We also seem to have 2 prop definitions - InputProps and InputWidgetProps. I think InputWidgetProps is the one to keep, just ensure all the properties we need are in that list (I don't think we need the `on...` events)